example.js

```diff
@@ -0,0 +1,43 @@
+// First let's import the logging module.
+var logging = require('./lib/log');
+
+// Now, let's create a module-level object.  The name
+// parameter is mandatory and should be the name of your
+// app.  The object passed in here is just a bunyan log
+// config object.
+var log = logging("my-example", {module: "main"});
```

Comment:
I'm confused... I see elsewhere that I'm not passing around `log`

So... is it crazy to create a bunyan logger for each file? ie. overhead? I doubt it's a problem, just curious...
Also we're doing this not to have to pass around `log` right?

----

okay, now just confused...